crates/miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -1265,43 +1266,56 @@ end
 #!
 #! Inputs:  []
 #! Outputs: []
-proc.compute_current_storage_commitment
-    # get number of storage slots
-    exec.memory::get_num_storage_slots
-    # => [num_storage_slots]
+proc.recompute_storage_commitment
```

Comment:
I would mention in the doc comments that we check if the commitment is out of date first.